infra/gcb/templates/src/build-status/build-status.html

```diff
@@ -191,23 +291,47 @@
       onTap(e) {
         // Change the route, this should auto-magically update the url in the
         // browser and invoke the _routeChanged method.
-        this.set('route.path', e.model.project.name);
+        this.set("route.path", e.model.project.name);
       }
 
       onChanged(e) {
-        if (this.$.build_type.selected == 'coverage') {
-          this.status = this.status_coverage
+        if (this.$.build_type.selected == "coverage") {
+          this.status = this.status_coverage;
         } else {
-          this.status = this.status_fuzzing
+          this.status = this.status_fuzzing;
         }
       }
 
-      showMessage(loading_log, log) {
-        return !loading_log && log === '';
+      onBuildHistoryClick(e) {
+        var ajax = this.$.logxhr;
+        const id = e.model.history.build_id;
+        const finishTime = e.model.history.finish_time;
+        ajax.url = "/log-" + id + ".txt";
+        ajax.generateRequest();
+        this.loading_log = true;
+        this.log = "";
+        this.finish_time = this.toLocalDate(finishTime);
+        this.build_id = id;
+      }
+
+      onLastSuccessClick(e) {
```

Comment:
This looks exactly the same as onBuildHistoryClick. Name this something more general and use in both on-click spots.